samples/CMakeLists.txt

```diff
@@ -32,7 +32,8 @@ function(add_chip_test EXEC_NAME TEST_NAME TEST_PASS SOURCE)
              )
 
     set_tests_properties("${TEST_NAME}" PROPERTIES
-             PASS_REGULAR_EXPRESSION "${TEST_PASS}")
+             PASS_REGULAR_EXPRESSION "${TEST_PASS}"
+             LABELS internal)
```

Comment:
what's the purpose internal?

Reply:
to be able to run a smaller subset of tests (`ctest -L internal`)